staging/src/k8s.io/apiserver/pkg/storage/value/encrypt/envelope/grpc_service_unix_test.go

```diff
@@ -127,6 +127,7 @@ func TestTimeouts(t *testing.T) {
 				kmsPluginWG     sync.WaitGroup
 				testCompletedWG sync.WaitGroup
 				socketName      = newEndpoint()
+				errc            = make(chan error, 1)
```

Comment:
if we use channel to propagate setup error then we don't need `kubeAPIServerWG` and  `kmsPluginWG` any longer.
```
kubeAPIErrCh, pluginErrCh   = make(chan error, 1), make(chan error, 1)
```

Reply:
make sense